Cangjie5.txt

```diff
@@ -70559,6 +70558,7 @@
 𢵟	qlkc
 𪭶	qll
 𫑢	qll
+𪮋	qll
```

Comment:
Although both 𪭶 and 𫑢 is rare, I place 𪮋 under them so people do not need retrain their muscle memory.

Reply:
You are right, but 𪮋 should be `qlll`. XD